templates/frontEndLayout.html

```diff
@@ -289,16 +299,218 @@
             <div  style = "text-align: left;">Cabbage</div>
         </td>
         <td>
-            <div class = "border">$2</div>
+            <div class = "border" onclick = "sell('c')">$2</div>
         </td>
         <td>
-            <div>2</div>
+            <div id = "ownedc">0</div>
         </td>
         <td>
-            <div class = "border">$12</div>
+            <div class = "border" onclick = "buy('c')">$12</div>
         </td>
     </tr>
 </table>
+<div id = "cash">CASH: $10</div>
+
+<script>
+var GAME_CONFIG = {
+    startingCash: 10,
+    seeds: {
+        a: {
+            name: "Alfalfa Sprouts",
+            image: "static/alfalfa.jpg",
+            imageSmall: "static/alfalfa.jpg",
+            imageMedium: "static/alfalfa.jpg",
+            imageLarge: "static/alfalfa.jpg",
+            buyCost: 3,
+            sellCost: 1,
+            harvestYield: 4,
+            harvestTimeSeconds: 3
+        },
+        b: {
+            name: "Broccoli",
+            image: "static/broccoli.jpg",
+            imageSmall: "static/broccoli.jpg",
+            imageMedium: "static/broccoli.jpg",
+            imageLarge: "static/broccoli.jpg",
+            buyCost: 6,
+            sellCost: 1,
+            harvestYield: 9,
+            harvestTimeSeconds: 120
+        },
+        c: {
+            name: "Cabbage",
+            image: "static/cabbage.jpg",
+            imageSmall: "static/cabbage.jpg",
+            imageMedium: "static/cabbage.jpg",
+            imageLarge: "static/cabbage.jpg",
+            buyCost: 12,
+            sellCost: 2,
+            harvestYield: 10,
+            harvestTimeSeconds: 300
+        }
+    }
+};
+
+var state = {
+    cash: GAME_CONFIG.startingCash,
+    a: 0,
+    b: 0,
+    c: 0,
+//    for(i = 0, i < 2, i++) {
+//      for(j = 0, j < 2, j++) {
+//        state.append("plot" + i + j);
+//            state["plot" + i + j].append(seedType: 0, timeLeft: 0);
+//        }
+//    }
+    plot00: {
+        seedType: 0,
+        timeLeft: 0
+    },
+    plot01: {
+        seedType: 0,
+        timeLeft: 0
+    },
+    plot02: {
+        seedType: 0,
+        timeLeft: 0
+    },
+    plot10: {
+        seedType: 0,
+        timeLeft: 0
+    },
+    plot11: {
+        seedType: 0,
+        timeLeft: 0
+    },
+    plot12: {
+        seedType: 0,
+        timeLeft: 0
+    },
+    plot20: {
+        seedType: 0,
+        timeLeft: 0
+    },
+    plot21: {
+        seedType: 0,
+        timeLeft: 0
+    },
+    plot22: {
+        seedType: 0,
+        timeLeft: 0
+    }
+};
+
+buy = function(seed) {
```

Comment:
when initializing a variable, you should precede it with `var `.  Chrome will figure it out if you don't, but it's good practice because to a reader it looks like you might have used `buy` previously and are just re-setting it here.